src/window.rs

```diff
@@ -682,6 +682,19 @@ impl Window {
     pub fn set_ime_position<P: Into<Position>>(&self, position: P) {
         self.window.set_ime_position(position.into())
     }
+
+    /// Requests user attention to the window, this has no effect if the application
+    /// is already focused. Providing `None` will unset the requesting of user attention.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Web / Wayland:** Unsupported.
+    ///
+    /// On macOS, providing `None` has no effect.
```

Comment:
Could you update platform specific section style so it'll look like the one you wrote for `RequestUserAttentionType` events, so it should be without a new line between `ios`/`macOS` and should be '- **MacOS:** `None` has no effect'.